config.js

```diff
@@ -206,6 +206,9 @@ var config = {
     // Enables support for opus-red (redundancy for Opus).
     // enableOpusRed: false,
 
+    // Enables support for opus-dtx where audio packets won’t be transmitted while participant is silent or muted.
+    // enableOpusDtx: false,
```

Comment:
This flag should go inside audioQuality, shouldn't it? We are slowly trying to move stuff into sections to avoid having a million settings.

Reply:
yeah, we were not sure, because of the opus red being outside... thanks!

Reply:
Moved to inside audioQuality.